warehouse/models/payments_views/payments_rides.sql

```diff
@@ -23,6 +23,7 @@
 " {{ create_row_access_policy(
     principals = ['serviceAccount:metabase@cal-itp-data-infra.iam.gserviceaccount.com',
                   'serviceAccount:bq-transform-svcacct@cal-itp-data-infra.iam.gserviceaccount.com',
+                  'serviceAccount:github-actions-services-accoun@cal-itp-data-infra.iam.gserviceaccount.com',
```

Comment:
Just want to confirm that we are aligned that giving this service account access to payments rides is preferable to refactoring to not use the compile-time variable?

Reply:
That would also be potentially preferable but I want the CI user to have some amount of access for eventual stats/testing; for example # of rows etc.

Reply:
Ok. Just want to confirm you've run this by the larger Payments group, I think it's related to that convo we had in the meeting last week.

Reply:
I'll ask, I hadn't yet

Reply:
I think the compile-time work isn't trivially replaced since it's used in the payments date spines and downstream models, but it's probably technically possible.

Reply:
FYI we got the OK to let the CI user have access since it's an internal system